src/components/Logo/Logo.tsx

```diff
@@ -59,40 +68,13 @@ export const Logo: React.FC<
             }}
         >
             {buttonIcon}
+            {!compact && logo}
         </Button>
     );
 
-    let logo: React.ReactNode;
-
-    if (typeof text === 'function') {
-        logo = text();
-    } else {
-        logo = (
-            <div className={b('logo')} style={{fontSize: textSize}}>
-                {text}
-            </div>
-        );
-    }
-
     return (
         <div className={b(null, className)}>
-            <div className={b('logo-btn-place', buttonWrapperClassName)}>
-                {hasWrapper ? wrapper(button, Boolean(compact)) : button}
-            </div>
-            {!compact &&
-                (hasWrapper ? (
-                    <div onClick={onClick}>{wrapper(logo, Boolean(compact))}</div>
-                ) : (
-                    <a
-                        href={href ?? '/'}
-                        target={target}
-                        rel={target === '_self' ? undefined : 'noreferrer'}
-                        className={b('logo-link')}
-                        onClick={onClick}
-                    >
-                        {logo}
-                    </a>
-                ))}
+            {hasWrapper ? wrapper(button, Boolean(compact)) : button}
```

Comment:
The button is not suitable for all cases. When clicking the button, unnecessary visual effects occur. In addition, there may be uses of wrappers that look at the node markup. In the major version, we plan to improve this component

Reply:
So, I've replaced the Button element with a native <a> tag. This removes the clicking visual effects and still keeps the element tabable.

This time, however, I had to retake a couple of screenshots (Footer and MobileFooter). There seems to be no visual difference in Storybook, including Safari, but there is a slight logo shift in Playwright screenshots for some reason.

Before: https://preview.gravity-ui.com/navigation/?path=/story/components-mobilefooter--showcase

After: https://preview.gravity-ui.com/navigation/305/?path=/story/components-mobilefooter--showcase